release_notes/374-release-notes.adoc

```diff
@@ -0,0 +1,321 @@
+
+Actions you need to take: Preemptively replace the `Policy` and `VulnerabilityReports` resources within your permission sets in favor of `WorkflowAdministration`.
+
+The permission `Access` will replace `Role` in permission sets in the {product-title-short} version 4.1 release. During the migration of the permission sets within the {product-title-short} version 4.1 release, the `Access` permission will have the lowest access permission granted for either `Access` or `Role`. For example, a permission set with READ Access and WRITE Role will have READ access after the migration. These changes in access can lead to unwanted side effects and missing access rights if you do not update your permission sets before the release.
+
+Actions you need to take: Preemptively replace the `Role` resource within your permission sets with `Access`. 
+
+The default `ScopeManager` role will be removed in the {product-title-short} version 4.1 release. During the migration, authentication provider rules referencing that role will be updated to use the `None` role.
+
+Actions you need to take: If authentication provider rules reference the `ScopeManager` role for other purposes than Vulnerability Report management, you should manually create a similar role and replace references to `ScopeManager` with the new role in the authentication provider rules.
+
+[id="in-product-docs-removal"]
+=== In-product docs removal
+
+With this release, Red Hat removed the in-product docs accessible from the help menu.
+If you are using the in-product docs, you can instead download the required documentation in PDF format from link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/[Red Hat Customer Portal]. (ROX-12839)
+
+[id="notice-support-upcoming-changes-374"]
+== Notice of upcoming support changes
+
+{product-title-short} version 3.74 is the last release that supports {ocp} versions 4.6, 4.7, 4.8, and 4.9. {product-title-short} version 4.0 will not support these {ocp} versions or earlier releases. See the link:https://access.redhat.com/node/5822721[Red Hat Advanced Cluster Security for Kubernetes Support Policy] for more information.
+
+[id="bug-fixes-374"]
+== Bug fixes
+
+[id="resolved-in-version-3740"]
+=== Resolved in version 3.74.0
+
+* The {product-title-short} Operator previously incorrectly removed owner references from the `stackrox-db` persistent volume claim (PVC). This can cause an error such as `Failed reconciling PVC "stackrox-db". Please remove the storageClassName and size properties from your spec [...]`. The bug is fixed. However, if you receive this message, you need to perform the steps described in the link:https://access.redhat.com/solutions/6995414[Knowledgebase article]. (ROX-14335)
```

Comment:
Since the ACS Jira isn't public, they would not come up for most readers, but the Product Team wants them in there for internal people who would be able to look up specific issues. Docs agreed to leave them in but unlinked.